### PR TITLE
Feature/login

### PR DESCRIPTION
**Pull request by andreihojda15**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ var passport = require("passport");
 require("dotenv").config();
 var gitHubRoutes = require("./routes/githubOAuth.js");
 var googleRoutes = require("./routes/googleOAuth.js");
+var loginRoutes = require('./routes/login.js');
```
Comment (denisslon-csv) [inline index.js:7]: You should use const instead of var on the three lines above.

File: routes/login.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+var express = require('express');
+var bcrypt = require('bcrypt');
```
Comment (denisslon-csv) [inline routes/login.js:2]: Use const instead of var here

File: index.js (hunk 1 above)
Comment (andreihojda15) [reply index.js:7]: changed

File: routes/login.js (hunk 2 above)
Comment (andreihojda15) [reply routes/login.js:2]: changed

File: index.js [hunk 3]
```diff
@@ -2,8 +2,9 @@ var express = require("express");
 var session = require("express-session");
 var passport = require("passport");
```
Comment (denisslon-csv) [inline index.js:3]: Those 3 should also be const

File: routes/login.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
+var express = require('express');
```
Comment (denisslon-csv) [inline routes/login.js:1]: Please use const here also

File: index.js (hunk 3 above)
Comment (andreihojda15) [reply index.js:3]: applied

File: routes/login.js (hunk 4 above)
Comment (andreihojda15) [reply routes/login.js:1]: changed

**Review by denisslon-csv — APPROVED** [review]

**Pull request closed without merging**